src/test/java/br/com/sbs/cubatech/dao/SubCategoryDaoTest.java

```diff
@@ -0,0 +1,90 @@
+package br.com.sbs.cubatech.dao;
+
+import br.com.sbs.cubatech.builder.CategoryBuilder;
+import br.com.sbs.cubatech.builder.SubCategoryBuilder;
+import br.com.sbs.cubatech.category.Category;
+import br.com.sbs.cubatech.category.SubCategory;
+import br.com.sbs.cubatech.category.SubCategoryDao;
+import br.com.sbs.cubatech.util.JPAUtilTest;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.persistence.EntityManager;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SubCategoryDaoTest {
+
+    private EntityManager entityManager;
+
+    private SubCategoryDao subCategoryDao;
+    private Category category;
+
+    @BeforeEach
+    void setUp() {
+        this.entityManager = JPAUtilTest.getEntityManager();
+        this.subCategoryDao = new SubCategoryDao(entityManager);
+        entityManager.getTransaction().begin();
+        this.category = aCategoryActive("Programação", "programacao");
+    }
+
+    @AfterEach
+    void tearDown() {
+        this.entityManager.getTransaction().rollback();
+    }
+
+    @Test
+    void shouldReturnListOfActiveSubCategories(){
```

Comment:
Somente sugestão:

```suggestion
    void getActiveSubCategory__should_return_active_categories() {
```
Na Alura seguimos uma padrão para criar os nomes dos métodos de testes, usamos o padrão `snake_case` e no começo dos testes adicionamos o nome do método que estamos testando

Reply:
Entendi. Atgé mais claro qual o método está sendo testado.